spec/system/placements/schools/school_contacts/user_adds_a_placement_contact_with_multiple_tabs_spec.rb

```diff
@@ -0,0 +1,206 @@
+require "rails_helper"
+
+RSpec.describe "Adding a placement contact to a school with multiple tabs", service: :placements, type: :system do
+  scenario "User adds a placement contact to their school using multiple tabs", :js do
+    given_i_am_signed_in
+
+    when_i_view_my_organisation_details_page_in_the_first_tab
+    then_i_do_not_see_any_placement_contact_details_in_the_first_tab
+
+    when_i_click_on_add_placement_contact_in_the_first_tab
+    and_i_fill_in_the_placement_contact_form_in_the_first_tab
+    and_i_submit_the_form_in_the_first_tab
+    then_i_see_the_check_your_answers_page_in_the_first_tab
+
+    when_i_view_my_organisation_details_page_in_the_second_tab
+    then_i_do_not_see_any_placement_contact_details_in_the_second_tab
+
+    when_i_click_on_add_placement_contact_in_the_second_tab
+    and_i_fill_in_the_placement_contact_form_in_the_second_tab
+    and_i_submit_the_form_in_the_second_tab
+    then_i_see_the_check_your_answers_page_in_the_second_tab
+
+    when_i_refresh_the_first_tab
+    then_the_school_placement_details_have_not_changed_in_the_first_tab
+
+    when_i_refresh_the_second_tab
+    then_the_school_placement_details_have_not_changed_in_the_second_tab
+
+    when_i_click_on_add_placement_contact_in_the_first_tab
+    then_i_see_my_placement_contact_details_in_the_first_tab
+    and_i_see_a_success_message_in_the_first_tab
+
+    when_i_click_on_add_placement_contact_in_the_second_tab
+    then_i_see_an_error_message_in_the_second_tab
+  end
+
+  private
+
+  def given_i_am_signed_in
+    @school = build(:placements_school, with_school_contact: false)
+    @windows = [open_new_window, open_new_window]
+    sign_in_placements_user(organisations: [@school])
+  end
```

Comment:
In terms of readability and explicitness, you might want to consider:

```suggestion
  def given_i_am_signed_in
    @school = build(:placements_school, with_school_contact: false)
    sign_in_placements_user(organisations: [@school])
  end
  
  def and_i_have_two_browser_tabs_open
    @windows = [open_new_window, open_new_window]
  end
```